src/jsModal.php

```diff
@@ -7,12 +7,61 @@
  */
 class jsModal extends jsExpression
 {
-    public function __construct($title, $url, $args = [], $mode = 'json')
+    public function __construct($title, $url, $args = [], $mode = 'json', $hasHeader = true)
     {
         if ($url instanceof VirtualPage) {
             $url = $url->getJSURL('cut');
         }
 
         parent::__construct('$(this).atkCreateModal([arg])', ['arg' => ['uri' => $url, 'title' => $title, 'mode' => $mode, 'uri_options' => $args]]);
+
+        if (!$hasHeader) {
```

Comment:
I think this can go to init() method or shouldn't this be before parent::construct?
Anyway having 5 parameters in constructor method call looks not well.